### PR TITLE
Fixes chameleon jumpsuits not updating properly when EMPd

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
Fixes chameleon jumpsuits not updating the user's appearance when EMPed and not properly reverting back to the default form when the EMP effect wears off.

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
While you're at it can you make it change to psychedelic jumpsuit when emagged? I had to adminbus my cham jumpsuit to get it to that because it's impossible to get

**Comment by Spacemanspark** [comment]
I miss the psychedelic jumpsuit.

File: code/modules/clothing/under/chameleon.dm [hunk 1]
```diff
@@ -39,11 +39,13 @@
 		desc = "Groovy!"
 		icon_state = "psyche"
 		item_color = "psyche"
+		usr.regenerate_icons()
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/chameleon.dm]: update the jumpsuit slot, don't regenerate literally everything =V

Comment (KasparoVy) [inline code/modules/clothing/under/chameleon.dm]: update_inv_w_uniform

File: code/modules/clothing/under/chameleon.dm [hunk 2]
```diff
@@ -39,11 +39,13 @@
...
 		spawn(200)
-			name = "Black Jumpsuit"
-			icon_state = "bl_suit"
-			item_color = "black"
-			desc = null
+			name = initial(name)
+			icon_state = initial(icon_state)
+			item_color = initial(item_color)
+			desc = initial(desc)
+			usr.regenerate_icons()
```
Comment (KasparoVy) [inline code/modules/clothing/under/chameleon.dm]: update_inv_w_uniform

File: code/modules/clothing/under/chameleon.dm [hunk 3]
```diff
@@ -39,11 +39,13 @@
 		desc = "Groovy!"
 		icon_state = "psyche"
 		item_color = "psyche"
+		usr.regenerate_icons()
 		spawn(200)
```
Comment (Iamgoofball) [inline code/modules/clothing/under/chameleon.dm]: >>>2016
>>>still using spawn()

no

Comment (Fox-McCloud) [inline code/modules/clothing/under/chameleon.dm]: Goofball is right, but I don't really expect you to fix this, in this PR.

Comment (Iamgoofball) [inline code/modules/clothing/under/chameleon.dm]: fix it anyways

**Pull request merged**